src/XRP/signer.ts

```diff
@@ -11,9 +13,9 @@ class Signer {
   /**
    * Encode the given raw JSON transaction to hex and sign it.
    *
-   * @param {TransactionJSON} transactionJSON The raw transaction JSON object.
-   * @param {Wallet} wallet The wallet to sign the transaction with.
-   * @returns {Uint8Array} A set of bytes representing the inputs and a signature.
+   * @param transactionJSON - The raw transaction JSON object.
+   * @param wallet  -The wallet to sign the transaction with.
```

Comment:
should there be a space both before and after the hyphen?

Reply:
Good catch! That's a typo on my end. This rule should really be auto-fixable, but the `tsdoc` plugin is in its early days, so I had to hand-fix these.